packages/core/src/components/TextField/Styled/InnerWrapper/outlined.styled.tsx

```diff
@@ -22,20 +22,26 @@ export const outlinedStyle = ({ theme, outlined, disabled, isErrorPresent }: Inn
             pointer-events: none;
             background-color: transparent;
             transition: all 100ms ease-out;
-            border-radius: ${theme.spacing.S1};
+            border-radius: ${outlined.default.borderRadius};
+            border-width: ${outlined.default.borderRadius};
             border: 0.1rem solid ${outlined.default.borderColor};
         }
         &:hover {
             box-shadow: ${!disabled && `0px 0.2rem 0.8rem ${rgba(outlined.hover.shadowColor, 0.2)}`};
         }
         &:focus-within {
+            border-radius: ${outlined.active.borderRadius};
             box-shadow: ${!disabled && `0px 0.2rem 0.8rem ${rgba(outlined[isErrorPresent ? 'error' : 'active'].shadowColor, 0.2)}`};
         }
 
-        &:hover::after,
-        &:focus-within::after {
-            border-width: ${!disabled && `0.15rem`};
+        &:hover::after {
+            border-width: ${!disabled && outlined.hover.borderWidth};
             border-color: ${!disabled && outlined.hover.borderColor};
         }
+        &:focus-within::after {
+            border-width: ${!disabled && outlined.active.borderWidth};
+            border-color: ${!disabled && outlined.active.borderColor};
+        }
```

Comment:
We can remove the `!disabled` here because this state isn't possible when disabled

Reply:
Also, should `border-radius` be included here as well?

Reply:
Cleaned up the `!disabled` check and added `border-radius`.